src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,96 @@
+.cards-catalog {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 0 auto;
+  width: getWidthCatalog(
+    $card-width,
+    $card-horisontal-shift,
+    $horisontal-shift
+  );
+  padding: $vertical-shift $horisontal-shift;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  background-color: $color-white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin-top: $card-vertical-shift;
+
+  &:not(:nth-child(4n)) {
+    margin-right: $card-horisontal-shift;
+  }
+
+  &:nth-child(-n + 4) {
+    margin-top: 0;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 3px;
+  }
+
+  &__description {
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 500;
+    color: $color-dark;
+    margin-top: 40px;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-light-grey;
+    margin-top: 4px;
+  }
+
+  &__price {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price-text {
+    font-weight: 400;
+    line-height: $price-line-height;
+    color: $color-light-grey;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: $price-line-height;
+    color: $color-dark;
+  }
+
+  &__link-buy {
+    text-align: center;
+    align-items: center;
+    text-transform: uppercase;
+    background-color: $color-blue;
+    border-radius: 5px;
+    text-decoration: none;
+    color: $color-white;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    padding: 12px 69px;
+    margin-top: 16px;
+
+    &:hover {
+      background-color: $color-white;
+      border: 1px solid $color-blue;
+      border-radius: 5px;
+      color: $color-blue;
+      padding: 11px 69px;
```

Comment:
why do we need to change a lot of styles on hover? change just background color and color
border radius and border is the same, just move the, to `&__link-buy`
and remove `padding` property